README.md

```diff
@@ -27,6 +27,12 @@ As early as possible in your application, require and configure dotenv.
 ```javascript
 require('dotenv').config()
 ```
+or 
```

Comment:
You could also specify that the `require('dotenv').config()` above is commonjs 🤔